test/fs/test_fs_js_api.c

```diff
@@ -111,6 +111,36 @@ int main() {
         assert(ex.name === 'ErrnoError' && ex.errno == 8 /* EBADF */);
     );
 
+    /********** test FS.utime() **********/
```

Comment:
I wonder if its worth splitting this function up.  (That way the ones that are JS-only can also be written using EM_JS instead of EM_ASM)

Reply:
(Can be a followup PR for that)

Reply:
Just to clarify, do you mean splitting up by moving each test section into an `EM_JS` function? Or should the code be split some other way?

Reply:
Yup thats what I meant.    Each section could be its own function (either a regular C function or an EM_JS function)